pkg/registry/registry.go

```diff
@@ -139,8 +139,7 @@ func (r *registry) ensureDeployment(appRegistries []string, needServiceAccount b
 				deployment.Spec.Template.Spec.ServiceAccountName = r.key.Name
 			}
 			// Update proxy env vars to match those in the operator.
-			deployment.Spec.Template.Spec.Containers[0].Env = proxy.GetInstance().GetEnvVars()
-
+			deployment.Spec.Template.Spec.Containers[0].Env = proxy.GetProxyEnvVars()
```

Comment:
Why not put the annotation on the registry deployments, too?

Reply:
The CVO team does not want us adding this annotation to our operands.